src/System.Private.CoreLib/shared/System/Globalization/GlobalizationMode.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Globalization
+{
+    internal static partial class GlobalizationMode
+    {
+        private const string InvariantModeConfigSwitch = "System.Globalization.Invariant";
+        internal static bool Invariant { get; } = GetGlobalizationInvariantMode();
```

Comment:
Is it really worth the ceremony to share 2 lines?

Reply:
Probably not but I didn't have time to refactor the platform-specific bits